src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
+  background: transparent;
```

Comment:
We don't usually reset tag's background to transparent as `none` is the default value for most of the tags, which may suit for more use cases.